### PR TITLE
OpenTelemetry: Improve support for long-running observable measurements

### PR DESCRIPTION
**Pull request by brunokoeferli**
* improve stability of AsyncObservableLongMeasurementTest.testLimitParallelExecution
* support OpenTelemetry attributes
* separate generic code from OpenTelemetry specific one (do not rely on Java extension mechanism)
* add leader election to execute measurements only on one cluster node

### CODE REVIEW

File: org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/opentelemetry/PeriodicAsyncMeasurement.java [hunk 1]
```diff
@@ -118,6 +122,10 @@ boolean requiresAsyncMeasurement() {
     return true;
   }
 
+  protected boolean isExecuteAsyncMeasurementOnThisNode() {
```
Comment (andibur) [inline org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/opentelemetry/PeriodicAsyncMeasurement.java]: Perhaps the name of this method as well as the supplier that provides the return value could be phrased more generally, such as `isActiveOnThisNode`.

**Review by andibur — APPROVED** [review]
Thanks

**Comment by brunokoeferli** [comment]
jenkins run tests

**Comment by brunokoeferli** [comment]
Pipeline successful: https://ci.eclipse.org/scout/job/org.eclipse.scout.rt_gerrit_jdk11_pipeline/2893/

**Pull request merged**